src/log.py

```diff
@@ -0,0 +1,55 @@
+"""
+Data Log
+
+By: Norma Chang
+"""
+import json
+import time
+
+def log(data, data_type):
+    """
+    Logs dictionary data from leader
+    """
+    if data_type == 'leader':
```

Comment:
What if I pass "LEADER?"

Reply:
What happens when I'm running this inside the follower but I pass "leader" as the vehicle type? And Vice versa.

Reply:
added error handling